fmn/consumer/worker.py

```diff
@@ -1,24 +1,23 @@
 # FMN worker figuring out for a fedmsg message the list of recipient and
 # contexts
 
-
 import json
 import logging
 import time
 import random
 
+from dogpile.cache import make_region
+from fedmsg_meta_fedora_infrastructure import fasshim
```

Comment:
somehow I thought the  order was along the lines of:
- modules from stdlib
- external modules
- app modules

and within these groups, ``import ..`` before, then ``from...import``.

I guess that is personal pref if it's not part of pep then.